<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>device_ac</class>
 <widget class="QWidget" name="device_ac">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame_background">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btn_xiao">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>410</y>
        <width>100</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_speed">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>350</y>
        <width>27</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_high">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>530</y>
        <width>100</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_auto">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>410</y>
        <width>100</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_mid">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>490</y>
        <width>100</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_ac">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>343</width>
        <height>240</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(186, 186, 186);</string>
      </property>
      <property name="text">
       <string>空调图片</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_dry">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>490</y>
        <width>100</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_low">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>450</y>
        <width>100</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_cold">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>450</y>
        <width>100</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_mode">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>350</y>
        <width>27</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_wind">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>530</y>
        <width>100</width>
        <height>35</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_text_ac">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>空调</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_onoff">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>72</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>ON</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_fenge">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>分割线</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
